examples/README.md

```diff
@@ -575,6 +576,61 @@ tutorial.
 
 [starccm-tutorial.yaml]: ../community/examples/starccm-tutorial.yaml
 
+### [batch-mpi.yaml] ![core-badge]
+
+This blueprint demonstrates how to use Spack to run a real MPI job on Batch.
+
+The blueprint contains the following:
+
+* A shared `filestore` filesystem.
+* A `spack-install` module that builts a script to install Spack and the WRF
+  application onto the shared `filestore`.
+* A `startup-script` module which uses the above script and stages job data.  
+* A builder `vm-instance` which performs the Spack install and then shuts down.
+* A `batch-job-template` that builds a batch job to execute the WRF job.
+* A `batch-login` VM that can be used to test and submit the Batch job.
+
+**Usage instructions:**
+
+1. Spack install
+
+    After `terraform apply` completes, you must wait for Spack installation to
+    finish before running the Batch job. You will observe that a VM named
+    `spack-builder-0` has been created. This VM will automatically shut down
+    once Spack installation has completed. When using a Spack cache this takes
+    about 25 minutes. Without a Spack cache this will take 2 hours. To view
+    build progress or debug you can inspect `/var/logs/messages` and
+    `/var/log/spack.log` on the builder VM.
+
+2. Access login node
+
+    After the builder shuts down, you can ssh to the Batch login node named
+    `batch-wrf-batch-login`. Instructions on how to ssh to the login node are
+    printed to the terminal after a successful `terraform apply`. You can
+    reprint these instructions by calling the following:
+
+    ```sh
+    terraform -chdir=batch-wrf/primary output instructions_batch-login
+    ```
+
+    Once on the login node you should be able to inspect the Batch job template
+    found in the `/home/batch-jobs` directory. This Batch job will call a script
+    found at `/share/wrfv3/submit_wrfv3.sh`. Note that the `/share` directory is shared between the login node and the Batch job.
```

Comment:
This line is suddenly not wrapped. (I realize its Markdown but while we're at it...)

Reply:
Done.